### PR TITLE
Create 2014-02-16 madelinecodingbat2.md

### PR DESCRIPTION
**Pull request by madeline13**

### CODE REVIEW

**Comment by eah13** [comment]
Madeline, this is on a `patch-3` branch instead of your `gh-pages` branch.

File: _posts/2014-02-16 madelinecodingbat2.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+author: madeline13
```
Comment (gerbal) [inline _posts/2014-02-16 madelinecodingbat2.md:3]: Make sure this is the same author name as in [config.yml](https://github.com/silshack/spring2014/blob/gh-pages/_config.yml#L208)

**Comment by eah13** [comment]
@madeline13 see me during the break in class today.

**Comment by eah13** [comment]
Closing this one; you're all straight now.

**Pull request closed without merging**